README.md

```diff
@@ -1,14 +1,28 @@
 ![Open Service Broker API](https://www.openservicebrokerapi.org/wp-content/uploads/2016/12/osbapi_logo_concept3_wtm.png)
 
 ## Latest Release: v2.12
+
 [Read the specification](https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md)
 
 [Release Notes for past versions](release-notes.md)
 
-## Working Changes
+## SDLC
+
+### Working Changes
```

Comment:
I see your point. Just because it's on Master doesn't mean it will go in the _next_ release, because we may be considering an earlier SHA of Master for the next release. 

I wonder if we can simply our release process to eliminate proposing a SHA and instead propose that we cut with whatever is on Master...